src/components/GameCards/GameCardText.vue

```diff
@@ -128,6 +138,18 @@ export default {
       }
     }
 
+    .game-completed {
+      padding: $gp / 8;
+      width: 25px;
+      height: 25px;
+
+      + .game-info {
```

Comment:
can the parent control this padding instead? this component shouldn't have to worry about styling a sibling